content/plugins/aws.md

```diff
@@ -8,36 +8,41 @@ weight: 100
 {{% gsSummary %}} {{% /gsSummary %}}
 
 
-# Description
+The AWS plugin allows users to use Cloudify to manage cloud resources on AWS. See below for currently supported resource types.
 
-The AWS plugin allows users to use Cloudify to manage cloud resources on AWS.
-For more information about the library, please refer to: [http://boto.readthedocs.org/en/latest/index.html](http://boto.readthedocs.org/en/latest/index.html).
+Be aware that some services and resources vary in availability between regions and accounts.
+
+For more information about the library, please refer [here](http://boto.readthedocs.org/en/latest/index.html).
 
 
 # Plugin Requirements
 
 * Python Versions:
   * 2.7.x
-* An account with AWS
-  * Your AWS [Access Keys](http://docs.aws.amazon.com/AWSSecurityCredentials/1.0/AboutAWSCredentials.html#)
+* AWS account
 
 
 # Compatibility
 
-The AWS EC2 plugin currently only supports classic EC2.
-
 The AWS plugin uses the following client libraries:
 
-  * [Boto client](https://github.com/boto/boto) - 2.34.0
+  * [Boto 2.38 client](https://github.com/boto/boto)
 
 {{% gsNote title="Note" %}}
 This version of Boto EC2 Connection supports (AWS) APIVersion = '2014-10-01'.
 {{% /gsNote %}}
 
 
+# Terminology
+
+* VPC is a virtual private cloud, for more info about VPCs refer to [AWS Documentation](https://aws.amazon.com/documentation/vpc/).
+* EC2-Classic is the original release of Amazon EC2. With this platform, instances run in a single, flat network that is shared with other customers.
+* `availability_zone` refers to the specific zone within a region, such as `us-west-1b`.  When specifying an `availability_zone`, you must specify one that is in the region you are connecting to.
+
+
 # Types
 
-The following are [node](reference-terminology.html#node) type definitions. Nodes describe cloud resources in your cloud infrastructure. For more information, see [nodes](reference-terminology.html#node).
+The following are [node](blueprints/spec-node-types/) type definitions. Nodes describe cloud resources in your cloud infrastructure. For more information, see [node](blueprints/spec-node-types/).
```

Comment:
please fix links according to the new structure - [node type]({{< relref "blueprints/spec-node-types.md" >}})

cloud resources -> resources